winpr/libwinpr/interlocked/test/TestInterlockedAccess.c

```diff
@@ -30,7 +30,7 @@ int TestInterlockedAccess(int argc, char* argv[])
 
 	if (*Addend != 10)
 	{
-		printf("InterlockedIncrement failure: Actual: %d, Expected: %d\n", (int) *Addend, 10);
+		printf("InterlockedIncrement failure: Actual: %ld, Expected: 10\n", (long) *Addend);
```

Comment:
Jup and currently LONG is not necessarily == long unfortunately  which I'm about to change with an upcoming PR